### PR TITLE
Add a sample layout to the readme.

### PR DESCRIPTION
**Pull request by chriseidhof**

### CODE REVIEW

**Comment by mattgallagher** [comment]
I reformatted the bodyText into a multiline literal during the merge. I wanted to use string interpolation to handle the italic sections but it wants `String` not `Text`. Unfortunately, the new SwiftUI localized string handling still doesn't play nicely with attributed strings (although it might be possible for a custom function to iterate through the `LocalizedStringKey.StringInterpolation` and apply formatting while building the underlying text).

Long story short though, I ditched the italics 🤕.
